tests/ops/op_math/test_sum.py

```diff
@@ -202,17 +202,14 @@ def test_ndim_params_raises_error(self):
         """Test that calling ndim_params raises a ValueError."""
         sum_op = Sum(qml.PauliX(0), qml.Identity(1))
 
-        with pytest.raises(
-            ValueError,
-            match="Dimension of parameters is not currently implemented for Sum operators.",
-        ):
+        with pytest.raises(AttributeError):
             _ = sum_op.ndim_params
 
     def test_batch_size_raises_error(self):
         """Test that calling batch_size raises a ValueError."""
         sum_op = Sum(qml.PauliX(0), qml.Identity(1))
 
-        with pytest.raises(ValueError, match="Batch size is not defined for Sum operators."):
+        with pytest.raises(AttributeError):
             _ = sum_op.batch_size
```

Comment:
I'm not too sure about these tests being required, still, but definitely fine to leave them in.
(It feels a bit like checking that a given operation does not have a decomposition or a sparse matrix defined)

Reply:
I generally don't like to delete tests.